### DISCUSSION TITLE
why Authentication user don't have created column?

### DISCUSSION THREAD

**Discussion by trongtri35** [supabase/supabase#5386] [General]
why Authentication user don't have created column? i need to know new resgister user like in firebase. 
Or i have to save myself in users table and query manually?

**Comment by GaryAustin1** [comment]
When a new user is added a ROW (you mention column, not sure you mean that) will be added to auth.users table.  You can see them in the UI or query that table if need be.
It is recommended you create a copy of the user data you want in a "public" table for general use if you need it.
See this for an idea:
https://supabase.com/docs/guides/auth/managing-user-data
